From a8ede9920ecd4ba61f48fb6a46cfe51ed457d68b Mon Sep 17 00:00:00 2001
From: NekoRouter <nekorouter@outlook.com>
Date: Fri, 11 Jul 2025 15:55:19 +0800
Subject: [PATCH 385/400] config: add console4A screen and DC charger support

Enable drm panel driver and gpio charger driver support for console4A.

Signed-off-by: NekoRouter <nekorouter@outlook.com>
---
 arch/riscv/configs/revyos_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/riscv/configs/revyos_defconfig b/arch/riscv/configs/revyos_defconfig
index f914ad108f15..81f3cf5adcbd 100644
--- a/arch/riscv/configs/revyos_defconfig
+++ b/arch/riscv/configs/revyos_defconfig
@@ -1128,6 +1128,7 @@ CONFIG_BATTERY_CW2015=y
 CONFIG_BATTERY_DS2760=m
 CONFIG_BATTERY_BQ27XXX=m
 # CONFIG_BATTERY_BQ27XXX_I2C is not set
+CONFIG_CHARGER_GPIO=y
 CONFIG_SENSORS_AD7414=m
 CONFIG_SENSORS_AD7418=m
 CONFIG_SENSORS_ADM1029=m
@@ -1450,6 +1451,7 @@ CONFIG_DRM_PANEL_ILITEK_ILI9881C=y
 CONFIG_DRM_PANEL_JADARD_JD9365DA_H3=y
 CONFIG_DRM_PANEL_HX8279=y
 CONFIG_DRM_PANEL_LT8911=y
+CONFIG_DRM_PANEL_MINGJUN_070BI30IA2=y
 CONFIG_DRM_ETNAVIV=m
 CONFIG_DRM_VERISILICON=y
 CONFIG_DRM_POWERVR_ROGUE=m
-- 
2.43.0

